src/harness/testcases/WINNF_FT_S_MCP_testcase.py

```diff
@@ -306,3 +315,227 @@ def test_WINNF_FT_S_MCP_1(self, config_filename):
        to 3700 MHz to protect configurable IAP-protected entities and DPAs
     """
     config = loadConfig(config_filename)
+
+    #config_parser = ConfigParser.RawConfigParser()
+    #config_parser.read(['sas_test_harness.cfg'])
+    #Headroom = config_parser.get('TestHarnessConfig', 'Headroom')
+    #ProtectionThreshold = config_parser.get('TestHarnessConfig', 'ProtectionThreshold')
+
+    cbsd_agg_list = []
+    fad_test_harnesses_objects = []
+    protected_entities = []
+    sas_test_harness_objects = []
+    domain_proxy_objects = []
+    for index in range(len(config['domainProxyConfigs'])):
+      domain_proxy_objects.append(sas_objects.DomainProxy(config['domainProxyConfigs'][index]['cert'], \
+                                                            config['domainProxyConfigs'][index]['key'], self))
+    # Step 1 : Load DPAs
+    self._sas_admin.TriggerLoadDpas()
+
+    # STEP 2 : ESC inf rms SAS about inactive DPA
+    self._sas_admin.TriggerBulkDpaActivation({'activate': False})
+
+    # Step 3 : creates multiple SAS TH, and Load predefined FAD,CBSD
+    for index, test_harness in enumerate(config['sasTestHarnessConfigs']):
```

Comment:
Can't you just use `test_harness` (which should be `test_harness_config`) directly instead of `config['sasTestHarnessConfigs'][index]`?

Reply:
Agreed, updated as suggested.